recipe/meta.yaml

```diff
@@ -18,12 +18,12 @@
 #       A Public Release for ISIS3.6.1:                        {% set version = "3.6.1" %}
 #       A Release Candidate for ISIS3.6.1:                     {% set version = "3.6.1_RC" %}
 #       A custom build of ISIS3.6.1 for the CaSSIS mission:    {% set version = "3.6.1_cassis" %}
-{% set version = "8.1.0_RC1" %}
+{% set version = "8.1.0_RC2" %}
 
 # This is the build number for the current version you are building. If this is the first build of
 # this version, the build number will be 0. It is incremented by 1 with every consecutive build of
 # the same version.
-{% set build_number = "0" %}
+{% set build_number = "1" %}
```

Comment:
I think this should be set to 0?

Reply:
Sounds good, does a consecutive build mean something like if I need to rebuild `8.1.0_RC1` and overwrite the existing RC1 conda package in Anaconda?

Reply:
Correct, in this case we are releasing a different version so we reset the build number to 0. If we went back to release RC1 for some reason we would increment the build number instead